pkg/triage/rule.go

```diff
@@ -121,11 +121,17 @@ func SummarizeRuleResult(t Rule, cs []*hubbub.Conversation, seen map[string]*Rul
 }
 
 // ExecuteRule executes a rule. seen is optional.
-func (p *Party) ExecuteRule(ctx context.Context, sp provider.SearchParams, t Rule, seen map[string]*Rule) (*RuleResult, error) {
+func (p *Party) ExecuteRule(ctx context.Context, sp provider.SearchParams, t Rule, seen map[string]*Rule, s *Collection) (*RuleResult, error) {
 	klog.V(1).Infof("executing rule %q for results newer than %s", t.ID, logu.STime(sp.NewerThan))
 	rcs := []*hubbub.Conversation{}
 	oldest := time.Now()
 
+	if s != nil {
+		if len(t.Repos) == 0 && len(s.Repos) > 0 {
```

Comment:
This isn't working for me because len(t.Repos) is never 0.  Rules seem to implicitly inherit the full list of repos from the top-level config.

Should I be configuring my rules with an explicitly-empty list of repos to allow the collection to override?  Or should the check on t.Repos be removed, and we have **any** collection scope **always** override rule scope?